pkg/ctl/utils/update_cluster_endpoint_access.go

```diff
@@ -0,0 +1,135 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/kris-nova/logger"
+	"github.com/spf13/pflag"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+)
+
+func updateEndpointAccessCmd(cmd *cmdutils.Cmd) {
+	cfg := api.NewClusterConfig()
+	cmd.ClusterConfig = cfg
+
+	cmd.SetDescription("update-cluster-api-access", "Update cluster API endpoint configuration", "")
+
+	cmd.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
+		cmdutils.AddNameFlag(fs, cfg.Metadata)
+		cmdutils.AddRegionFlag(fs, cmd.ProviderConfig)
+		cmdutils.AddConfigFileFlag(fs, &cmd.ClusterConfigFile)
+		cmdutils.AddApproveFlag(fs, cmd)
+		cmdutils.AddTimeoutFlag(fs, &cmd.ProviderConfig.WaitTimeout)
+	})
+
+	var private, public bool
+	cmd.FlagSetGroup.InFlagSet("Allow/Disallow Cluster API access flags",
+		func(fs *pflag.FlagSet) {
+			fs.BoolVar(&private, "private-access", false, "access for private (VPC) clients")
+			fs.BoolVar(&public, "public-access", true, "access for public clients")
+		})
+	cmd.SetRunFuncWithNameArg(func() error {
+		return doConfigureEndpointAccess(cmd, private, public)
+	})
+
+	cmdutils.AddCommonFlagsForAWS(cmd.FlagSetGroup, cmd.ProviderConfig, false)
+}
+
+func doConfigureEndpointAccess(cmd *cmdutils.Cmd, newPrivate bool, newPublic bool) error {
```

Comment:
based on current convention this should be called `do<Action>`

Reply:
Isn't it that way, where ConfigureEndpointAccess is the action?  Or should it be doConfiguredClusterEndpointAccess?

Reply:
changed.